open-metadata-resources/open-metadata-deployment/charts/vdc/Chart.yaml

```diff
@@ -1,7 +1,8 @@
 # <!-- SPDX-License-Identifier: Apache-2.0 -->
```

Comment:
Worth dropping the XML comment surround here as well?

Reply:
Agreed - cut/paste error.
Corrected and repushed.